### ISSUE TITLE
[BUG] Bastille is reporting incorrect installed version

### ISSUE THREAD

**Issue by jemail99** [BastilleBSD/bastille#753]
**[MANDATORY] Describe the bug [MANDATORY]**
bastille reports incorrect installed version

**[MANDATORY] Bastille and FreeBSD version (paste ``bastille -v && freebsd-version -kru`` output)**
```
bastille -v && freebsd-version -kru
0.10.20231125
14.1-RELEASE
14.1-RELEASE
14.1-RELEASE
```

**[MANDATORY] How did you install bastille? (port/pkg/git)**
Installed using pkg

Details:
I am on 14.1 using latest repositories but you will note the above bastille version reported is 0.10.20231125.
```
pkg -vv
....
Repositories:
FreeBSD: {
url             : "**pkg+http://pkg.FreeBSD.org/FreeBSD:14:amd64/latest**",
enabled         : yes,
priority        : 0,
mirror_type     : "SRV",
signature_type  : "FINGERPRINTS",
fingerprints    : "/usr/share/keys/pkg"
```

Pkg info shows:
```
$ pkg info
bastille-0.11.20241022         Jail automation framework
.....
```

Reinstalling produces:
```
pkg install bastille-0.11.20241022
Updating FreeBSD repository catalogue...
FreeBSD repository is up to date.
All repositories are up to date.
Checking integrity... done (0 conflicting)
The most recent versions of packages are already installed
```
**[optional] Expected behavior**
Bastille installed from latest repositories shows version 0.11.20241022 when using bastille -v

**Comment by tschettervictor** [comment]
Looks like we forgot to change the version inside "common.sh"

The version is the 0.11.20241022 though.

**Comment by yaazkal** [comment]
@jemail99 thanks, this is a duplicate and has been already resolved in #728, but a newer version has been released after that. Actually the latest version is [0.12.20241124](https://github.com/BastilleBSD/bastille/releases/tag/0.12.20241124) but I'm not sure if that version has landed yet in ports, You can download it from this github repo anyway.

Regards.
